container-search/src/main/java/com/yahoo/prelude/searcher/FieldCollapsingSearcher.java

```diff
@@ -29,8 +29,8 @@
 public class FieldCollapsingSearcher extends Searcher {
 
     private static final CompoundName collapse = new CompoundName("collapse");
-    private static final CompoundName collapsefield=new CompoundName("collapsefield");
-    private static final CompoundName collapsesize=new CompoundName("collapsesize");
+    private static final CompoundName collapsefield=new CompoundName("collapse.field");
+    private static final CompoundName collapsesize=new CompoundName("collapse.size");
```

Comment:
You'll need to add this to the query profile type in Query such that you can set the old names as aliases.

Reply:
Can I make the CompoundNames public, so I can refer to them in Query, when adding them to the query profile type?

Reply:
Yes.